plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -39,6 +40,19 @@ class ServiceModals extends React.Component {
     );
   }
 
+  getCannotDestroyGroupModal(groupFrameworks) {
```

Comment:
Also please align this method with the revised name for the modal :)

Reply:
updated